lightning/src/ln/msgs.rs

```diff
@@ -2277,13 +2319,17 @@ impl Writeable for OutboundOnionPayload {
 			},
 			Self::Receive {
 				ref payment_data, ref payment_metadata, ref keysend_preimage, amt_msat,
-				outgoing_cltv_value, ref custom_tlvs,
+				outgoing_cltv_value, ref trampoline_packet, ref custom_tlvs
 			} => {
 				// We need to update [`ln::outbound_payment::RecipientOnionFields::with_custom_tlvs`]
 				// to reject any reserved types in the experimental range if new ones are ever
 				// standardized.
 				let keysend_tlv = keysend_preimage.map(|preimage| (5482373484, preimage.encode()));
-				let mut custom_tlvs: Vec<&(u64, Vec<u8>)> = custom_tlvs.iter().chain(keysend_tlv.iter()).collect();
+				let trampoline_tlv = trampoline_packet.as_ref().map(|trampoline| (66100, trampoline.encode()));
```

Comment:
Out of curiosity, I was able to find a reference for why keysend why keyed by `5482373484` in [blip-0003](https://github.com/lightning/blips/blob/master/blip-0003.md), but I was able to find no such reference for `66100`, for the trampoline. Is it an arbitrary choice for now, or is there some reasoning behind it?

Reply:
Thanks a lot for the reference!

I think it will be a good idea to write a comment about it, explaining the reasoning behind it and the necessary caution one should keep.
Eclair's comment seems like a pretty solid one to take a reference from.

> // Types below aren't specified - use cautiously when deploying (be careful with backwards-compatibility).